### PR TITLE
Feature - Border for Tooltip (Issue: #692)

### PR DESCRIPTION
**Pull request by emelinepal**
Add tooltipBorder property in LineTouchTooltipData, BarTouchTooltipData, ScatterTouchTooltipData

### CODE REVIEW

**Comment by imaNNeoFighT** [comment]
Hi. I think you need to update your branch with our `master` branch.

File: repo_files/documentations/bar_chart.md [hunk 1]
```diff
@@ -158,4 +159,4 @@ enum values {`start`, `end`, `center`, `spaceEvenly`, `spaceAround`, `spaceBetwe
 
 
 ##### Gist - Toggleable Tooltip ([Source Code](https://gist.github.com/imaNNeoFighT/bce3f0169ff3fd6c3f137cdeb5005c0e))
-https://user-images.githubusercontent.com/7009300/156784816-53f95dd9-f387-4600-8a92-d05b1aeea3da.mov
```
Comment (imaNNeoFighT) [inline repo_files/documentations/bar_chart.md:161]: This is a redundant change. Can you please revert it?

File: repo_files/documentations/scatter_chart.md [hunk 2]
```diff
@@ -49,6 +49,7 @@ When you change the chart's state, it animates to the new state internally (usin
 |PropName|Description|default value|
 |:-------|:----------|:------------|
 |tooltipBgColor|background color of the tooltip bubble|Colors.white|
+ |tooltipBorder|border of the tooltip bubble|BorderSide.none|
```
Comment (imaNNeoFighT) [inline repo_files/documentations/scatter_chart.md:52]: There is an extra space here :)

**Comment by imaNNeoFighT** [comment]
Congratulations @emelinepal.
There are just two minor comments. Please check them out.
You did it very smoothly. It seems you read the contribution guideline carefully, and I appreciate it.
Please resolve those two minor discussions, then make sure to update your branch with the `master` 

Thanks!

File: repo_files/documentations/bar_chart.md (hunk 1 above)
Comment (emelinepal) [reply repo_files/documentations/bar_chart.md:161]: Hello @imaNNeoFighT  ! 
I'm not sure what happened here as the two lines are the same 🤔
I don't know how can I revert it ? 😖

Comment (imaNNeoFighT) [reply repo_files/documentations/bar_chart.md:161]: You can edit your latest commit using `--amend` flag. Like the below command:
`git commit --amend --no-edit` (`--no-edit` is for when you don't want to edit the message, you just need to add your staged changes to the latest commit without changing the commit message).

Also, you can edit the previous commits (before the latest commit) using `interactive rebase` which is an advanced topic. You can find more about it if you google it.

**Comment by imaNNeoFighT** [comment]
Still, I cannot merge it. It has some conflicts with the `master` branch.
Screenshot from 2022-06-12 00-59-20

**Comment by imaNNeoFighT** [comment]
Continued on #1060

**Pull request closed without merging**